spotbugs/etc/messages.xml

```diff
@@ -4544,6 +4544,16 @@ it may indicate a misunderstanding of how serialization works.
   <p> This class contains a field that is updated at multiple places in the class, thus it seems to be part of the state of the class. However, since the field is marked as transient and not set in readObject or readResolve, it will contain the default value in any
 deserialized instance of the class.
 </p>
+]]>
+    </Details>
+  </BugPattern>
+  <BugPattern type="SE_PREVENT_EXT_OBJ_OVERWRITE">
+    <ShortDescription>Prevent overwriting of externalizable objects</ShortDescription>
+    <LongDescription>Any caller can reset the value of the object by using the readExternal() method.</LongDescription>
+    <Details>
```

Comment:
This doesn't say anything about how the issue might be resolved.

Reply:
I add a solution suggestion section to the description.

Reply:
> This doesn't say anything about how the issue might be resolved.

Are you good with the proposed solution?  I think it reads ok generally but what are your thoughts?

@Spatrik95 I fixed the change log merge conflict for you.  If @ThrawnCA is good I Think we can merge in.

Reply:
I went ahead and merged.  Thanks.